src/core/MRSystem.js

```diff
@@ -145,9 +142,9 @@ export class MRSystem {
      * @description Handles the component and registry aspect of the event when an entity component attaches to this system.
```

Comment:
can you add `(async)` to the description like you did for the other newly async functions?